src/api/lib/tasks/dev/test_data/maintenance.rb

```diff
@@ -131,6 +131,26 @@ def mimic_patchinfo(project, comment = 'test comment')
       Patchinfo.new.create_patchinfo(project, 'patchinfo', comment: comment)
     end
 
+    # Simulate `osc maintenancerequest servers apache2 openSUSE:Leap:15.4:Update`
+    def mimic_maintenancerequest(source_project_name:, source_package_name:, target_project_name:, target_releaseproject_name:)
```

Comment:
We've realized this method can be merged with `create_request_with_incident_actions`. We are going to do it, so this PR will be moved to Draft for a while